solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -39,8 +39,8 @@ contract StandardPoolConverter is
     uint32 private constant PPM_RESOLUTION = 1000000;
 
     uint256 private __reserveBalances;
-    IERC20Token[] private __reserveTokens;
-    mapping(IERC20Token => uint256) private __reserveIds;
+    IERC20Token[] internal __reserveTokens;
```

Comment:
these two shouldn't be internal either, __reserveTokens can be accessed using the gettre, ids are part of the internal implementation of the standard pool

Reply:
Fixed.

Reply:
I don't see that change

Reply:
Fixed.